packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -894,6 +894,11 @@ new tasks.SageMakerCreateTransformJob(this, 'Batch Inference', {
   transformResources: {
     instanceCount: 1,
     instanceType: ec2.InstanceType.of(ec2.InstanceClass.M4, ec2.InstanceSize.XLARGE),
+  },
+  dataProcessing: {
+    inputFilter: '$[1:]',
+    joinSource: 'Input',
+    outputFilter: '$[0,-1]'
```

Comment:
the thing that stands out of this example is that this sampe code is cryptic. perhaps we need to add utility methods and enums to make it clearer what it's actually doing.

at a minimum please add some comments explaining what's setting these properties accomplishes.